src/lib.rs

```diff
@@ -226,6 +278,7 @@ impl<'a> Chart<'a> {
             y_style: LineStyle::Dotted,
             x_label_format: LabelFormat::Value,
             y_label_format: LabelFormat::Value,
+            y_axis_tick_labels: YAxisTickLabels::None,
```

Comment:
You can omit the word `axis` in the name

Reply:
Done. What do you think about the name of the enum `YAxisTickLabels`?

Reply:
Right.. `TickDisplay` or `TickDisplayStyle`, wdyt?

Reply:
I changed it to `TickDisplay`. Thought that was more in line with the codebase style.